spec/views/posts/show.html.erb_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "posts/show", type: :view do
+  before(:each) do
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] RSpec/HookArgument: Omit the default :each argument for RSpec hooks.